TECH-LEADS-CHECKLIST.md

```diff
@@ -40,12 +40,13 @@ More details on [Unit Testing](test-first-development/unit-testing/readme.md)
 
 ## Observability
 
+- [ ] Significant business and functional events are tracked and related metrics collected.
 - [ ] Application faults and errors are logged.
-- [ ] Metrics collected to identify health of the services.
-- [ ] Latency for servicing a request is logged, as appropriate for the solution.
-- [ ] Significant business and functional events that are important are tracked and related metrics collected.
-- [ ] Logging configuration can be modified without code changes (eg: debug mode).
-- [ ] GDPR compliance is ensured regarding PII (personal identifiable information).
+- [ ] Health of the system is monitored.
+- [ ] Make sure the client and server side observability data can be differentiated.
```

Comment:
Suggestion to follow the format of the other items:
Make sure the client and server side observability data can be differentiated -> The client and server side observability data can be differentiated or alternatively something like Client and server side logs can be collected separately